mini-wallet.md

```diff
@@ -52,86 +50,106 @@ Options:
+	                                  				
+	 -f, --faucet TEXT               					Testnet faucet URL.  [default:
+	                                 				  http://testnet.diem.com/mint]
+	                                 				  
+	 --pytest-args TEXT              					Additional pytest arguments, split by empty
+	                                  				space, e.g. `--pytest-args '-v -s'`
+	                                  				
+	 -d, --test-debug-api BOOLEAN    					Run tests for debug APIs.  [default: False]
+	 
+	 -v, --verbose                   					Enable verbose log output.  [default: False]
+	 
+	 --help                          					Show this message and exit.
+	```
 
-## MiniWallet Test Suite
-
-Try out the MiniWallet Test Suite by hitting the target server we started by `dmw start-server`
-```
-dmw test --target http://localhost:8888
-```
-You should see something like this:
-
-```
-> dmw test
-Diem JSON-RPC URL: http://testnet.diem.com/v1
-Diem Testnet Faucet URL: http://testnet.diem.com/mint
-======================================== test session starts ===============================================
-…...
-collected 61 items
 
-src/diem/testing/suites/test_miniwallet_api.py ......................ss.                              [ 40%]
-src/diem/testing/suites/test_payment.py ....................................                          [100%]
 
-============================== 59 passed, 2 skipped, 198 deselected in 18.26s ===============================
-```
-
-Follow the MiniWallet API specification to create a proxy server for your wallet application.
-Assuming you're running your application at port 9999, you may run the MiniWallet Test Suite using this command:
-```
-dmw test --target http://localhost:9999
-```
-
-`test` options:
-```
-dmw test --help
-Usage: dmw test [OPTIONS]
-
-Options:
-  -t, --target TEXT               Target mini-wallet application URL.
-                                  [default: http://localhost:8888]
-
-  -h, --stub-bind-host TEXT       The host the miniwallet stub server will
-                                  bind to  [default: localhost]
-
-  -p, --stub-bind-port INTEGER    The port the miniwallet stub server will
-                                  bind to. Random if empty.
+### How it works
 
-  -u, --stub-diem-account-base-url TEXT
-                                  The address that will be used for offchain
-                                  callbacks. Defaults to
-                                  http://localhost:{random_port}
+The `diem.testing.suites` package contains the Diem MiniWallet Test Suite and includes the pytest file `conftest.py`.
 
-  -j, --jsonrpc TEXT              Diem fullnode JSON-RPC URL.  [default:
-                                  http://testnet.diem.com/v1]
-
-  -f, --faucet TEXT               Testnet faucet URL.  [default:
-                                  http://testnet.diem.com/mint]
+Launch a pytest runner to run tests using:
```

Comment:
"`dmw test` launches a pytest runner to run tests." tells what `dmw test` is going to do.
I don't think above "Launch a pytest runner to run tests using: `dmw test`" has same meaning. `dmw test` is not the only way you can launch the test suite.

Reply:
Changed language to -  Use `dmv test` to launch pytest runner to run tests - to clarify that dmv test launches the pytest runner and does not assume that this is the only way to launch the test suite.